### DISCUSSION TITLE
Notification sounds in flet

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#3240] [Q&A] [answered]
### Question

Good morning. Is there any way to have notification sounds in Flet? for example, if a new message is coming, or a notification for a task or meeting or something like that? Is a new control needed or is it possible to do it with the existing controls? Thanks

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
You can use the [`Audio`](https://flet.dev/docs/controls/audio) control to play sound on any device.
